config.json

```diff
@@ -2465,6 +2465,21 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "a68113f8-92be-40e0-a97f-eb4239303ef5",
+        "prerequisites": [
+          "cond",
+          "case",
+          "if",
+          "erlang-libraries"
+        ],
+        "practices": [
+          "cond"
+        ],
+        "difficulty": 2
+      },
```

Comment:
@neenjaw Did we agree recently that `Float.pow` and `Integer.pow` (added in Elixir 1.12) count as part of the integers and floats?

If that's the case, then we could say that this is a cond + integers + floats exercise (although we might need leaving a hint that sqrt(x) == pow(x, 0.5) for less math-savvy students).

We have a severe lack of very easy exercises that could be done after solving only very few learning exercises. If we made that decision, then this exercise will actually become the very first exercise that people learning Elixir can solve. All it takes it that they solve `lasagna` (basics), `freelancer-rates` (integers), `pacman-rules` (booleans) and `log-level` (cond)


For reference for @jiegillet (not sure if you have been on the staging website exercism.lol (sic)), here's our current concept tree (on the website you can browse it better than on a screenshot):

![elixir concepts](https://user-images.githubusercontent.com/7852553/123283204-750bdb80-d50b-11eb-9613-4e48a0f5255f.png)

Reply:
OK, let's leave out `erlang-libraries`. 
Technically there is also a solution that doesn't rely on square roots at all. Something like 
```elixir
radius_sq = x * x + y * y
cond do
  radius_sq > 10 * 10
  ...
```
Although I don't think less math-savvy students would think of that ^^